### DISCUSSION TITLE
PLEASE HELP

### DISCUSSION THREAD

**Discussion by Priii-13** [GarimaSingh0109/WasteManagment#83] [Q&A]
i want to work on the issue... i am new to open source contributions.. in this project the issues are raised by other than admins... how can we work on issues ????

**Comment by GarimaSingh0109** [comment]
Hii @Priii-13 you can create your own issue make the issue title short & to the point and in issue description write briefly about the issue like What is the issue How is it impacting the website Benefits of improving the issue (try writing in pointers) ss of issue
After issue is assigned work on it if facing any issue ask in discussion here (like you can create a new one) 
Create PR and write in the same above format add the ss of you work(not code)(before & after)
After you create a pr i ll review ur code and merge it if i want some changes or some errors are there you will be informed
If you have any further suggestion you can definatly reach out to us you can also get some browniee points
